VHanna/HW6/HW6_task2.py

```diff
@@ -0,0 +1,11 @@
+login = 'First'
```

Comment:
good job
look at this version of code

login = 'First'
max_attempts = 3
attempts = 0

while True:
    user_login = input('Please input your login (or type "exit" to quit): ')
    
    if user_login.lower() == 'exit' or user_login.lower() == 'quit':
        print('Exiting program.')
        break
    
    if user_login == login:
        print(f'Hi {login}!')
        break
    else:
        attempts += 1
        print('Wrong login. Please try again.')

        if attempts >= max_attempts:
            print('Exceeded maximum attempts. Exiting.')
            break